examples/seq2seq/finetune.py

```diff
@@ -279,16 +269,13 @@ def add_model_specific_args(parser, root_dir):
         parser.add_argument("--freeze_encoder", action="store_true")
         parser.add_argument("--freeze_embeds", action="store_true")
         parser.add_argument("--sortish_sampler", action="store_true", default=False)
-        parser.add_argument("--logger", type=str, choices=["default", "wandb", "wandb_shared"], default="default")
+        parser.add_argument("--logger_name", type=str, choices=["default", "wandb", "wandb_shared"], default="default")
         parser.add_argument("--n_train", type=int, default=-1, required=False, help="# examples. -1 means use all.")
         parser.add_argument("--n_val", type=int, default=500, required=False, help="# examples. -1 means use all.")
         parser.add_argument("--n_test", type=int, default=-1, required=False, help="# examples. -1 means use all.")
         parser.add_argument(
             "--task", type=str, default="summarization", required=False, help="# examples. -1 means use all."
         )
-        parser.add_argument("--src_lang", type=str, default="", required=False)
```

Comment:
this is causing the mbart failure.
I assume this is all a big git accident.
If you are in pycharm, you can checkout master, fetch upstream, checkout your branch and hit cmd-K to easily put back things you accidentally deleted.